package.json

```diff
@@ -72,7 +73,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">=4.2.x"
+    "node": ">=6.9.x"
```

Comment:
Let's avoid this change until 5.x

Out of curiosity, what in this PR depends on it?